twake/backend/node/src/core/platform/services/search/adapters/elasticsearch/index.ts

```diff
@@ -252,9 +252,13 @@ export default class ElasticSearch extends SearchAdapter implements SearchAdapte
     };
 
     let esResponse: ApiResponse;
+
     if (options.pagination.page_token) {
       esResponse = await this.client.scroll(
-        { scroll_id: options.pagination.page_token, ...esParamsWithScroll },
+        {
+          scroll_id: options.pagination.page_token,
+          // ...esParamsWithScroll
```

Comment:
Could you explain to me why is it removed?
And then depending on the response I suppose there is two option here:
- Put it back
- Remove it completely
😛

Reply:
It doesn't work with query in `_search/scroll`. It expects only `scroll_id` that refers to the previous query, and it falls with error if query exists in this request. I tested on a local elastic search and found this pattern. The documentation mentions it in passing, but it doesn't work with `query` anyway. So I suppose we can remove it completely.